checker/tests/optional/JdkCheck.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Optional;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.checker.optional.qual.Present;
+
+/** Test JDK annotations. */
+@SuppressWarnings("optional.parameter")
+public class JdkCheck {
+
+    boolean isPresentTest1(@Present Optional<String> pos) {
+        return pos.isPresent();
+    }
+
+    boolean isPresentTest2(Optional<String> mos) {
+        return mos.isPresent();
+    }
+
+    // No JDK annotations on methods using Java 8 types yet.
```

Comment:
I'm rebuilding the JDK again.  Maybe these can be uncommented after that is done?

Reply:
Yes, I think they can.  Could you please try that when you get a chance this morning?